src/core_modules/capture-core/components/LockedSelector/LockedSelector.container.js

```diff
@@ -66,5 +78,9 @@ const mapDispatchToProps = (dispatch: ReduxDispatch, { customActionsOnProgramIdR
     },
 });
 
-// $FlowFixMe[missing-annot] automated comment
-export const LockedSelector = connect(mapStateToProps, mapDispatchToProps)(withLoadingIndicator()(LockedSelectorComponent));
+export const LockedSelector: ComponentType<OwnProps> =
```

Comment:
I also took a moment and annotated this one since it was within the code I touched.